test/integration/files.test.ts

```diff
@@ -136,4 +137,39 @@ describe('file support', () => {
     const content = await attachment?.text();
     expect(content).toBe('hello,world');
   });
+
+  test('create XataFile with upload signed url', async () => {
+    const result = await xata.db.users.create(
+      {
+        name: 'Test',
+        photo: { name: 'pic.png' },
+        attachments: [{ name: 'hello.csv', mediaType: 'text/csv' }]
```

Comment:
```suggestion
        photo: { name: 'pic.png', base64Content: '' },
        attachments: [{ name: 'hello.csv', mediaType: 'text/csv', base64Content: '' }]
```

I have asked @sorintm to remove the `base64Content` as required, but right now it fails with:

```
{ status: 400, errors: [ { message: 'invalid record: column [attachments]: file entry [0]: invalid \'base64Content\': value is required', status: 400 } ], requestId: '4808bc1d-38fb-9157-801c-7c7a05f994cc' }
```